assets/styles/app.scss

```
@@ -2,3 +2,15 @@
 label.required::after {
     content: ' *';
 }
+
+:root {
+    --x-icon-decorative-blue-france: #CACAFB;
+}
+
+.fr-x-icon-decorative--blue-france::before {
+    background-color: var(--x-icon-decorative-blue-france);
+}
+
+.fr-icon--xl::before {
```

(Pinaillage) Vu qu'on étend le DSFR ici aussi ?

```suggestion
.fr-x-icon--xl::before {
```

---

Ok je modifie